ui/app/helpers/constants/transactions.js

```diff
@@ -23,3 +24,12 @@ export const ENCRYPTION_PUBLIC_KEY_REQUEST_KEY = 'encryptionPublicKeyRequest'
 export const CONTRACT_INTERACTION_KEY = 'contractInteraction'
 export const CANCEL_ATTEMPT_ACTION_KEY = 'cancelAttempt'
 export const DEPOSIT_TRANSACTION_KEY = 'deposit'
+export const INCOMING_TRANSACTION_KEY = 'incoming'
```

Comment:
Nit: Calling this a "key" is a bit misleading, because this isn't the key for any localized messages.

It's quite unfortunate how these constants were originally written - first as keys to localized messages about specific transaction categories, then later re-used as transaction category was added as a real `txMeta` property. Also the pattern of using variables as the parameter to the translation method makes static analysis of localized messages much more difficult. Ah well.

For now I'd suggest removing `_KEY` from this variable name, and leaving the rest as-is. But at some point I'd like to return here and stop using these as translation keys altogether, instead using these just as transaction category enums.

Reply:
I wasn’t even quite sure what _KEY was referring to here and didn’t pick up the significance of it mapping to a localization key. I was attempting to keep the patterns of the file in place but I don’t have any objections to your request.

Reply:
I believe this is resolved now.